microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode.md

```diff
@@ -102,41 +107,41 @@ By default, the **Activity Bar** contains the following modes and tools:
 
 *  **Device Emulation** mode (![Device Emulation icon](../media/device-emulation-icon-light-theme.png)) button.
 
-*  **Welcome** tool (![Welcome tool icon](media/focus-mode-welcome.png)).
+*  **Welcome** tool (![Welcome tool icon](./focus-mode-images/welcome.png)).
 
-*  **Elements** tool (![Elements tool icon](media/focus-mode-elements.png)).
+*  **Elements** tool (![Elements tool icon](./focus-mode-images/elements.png)).
 
-*  **Console** tool (![Console tool icon](media/focus-mode-console.png)).
+*  **Console** tool (![Console tool icon](./focus-mode-images/console.png)).
 
-*  **Sources** tool (![Sources tool icon](media/focus-mode-sources.png)).
+*  **Sources** tool (![Sources tool icon](./focus-mode-images/sources.png)).
 
-*  **Network** tool (![Network tool icon](media/focus-mode-network.png)).
+*  **Network** tool (![Network tool icon](./focus-mode-images/network.png)).
 
-*  **Issues** tool (![Issues tool icon](media/focus-mode-issues.png)).
+*  **Issues** tool (![Issues tool icon](./focus-mode-images/issues.png)).
 
-*  **Performance** tool (![Performance tool icon](media/focus-mode-performance.png)).
+*  **Performance** tool (![Performance tool icon](./focus-mode-images/performance.png)).
 
-*  **Memory** tool (![Memory tool icon](media/focus-mode-memory.png)).
+*  **Memory** tool (![Memory tool icon](./focus-mode-images/memory.png)).
 
-*  **Application** tool (![Application tool icon](media/focus-mode-application.png)).
+*  **Application** tool (![Application tool icon](./focus-mode-images/application.png)).
 
 
 <!-- ====================================================================== -->
 ## Pin and rearrange tools in the Activity Bar
 
 You can choose which tools appear in the **Activity Bar** by pinning or unpinning tools. This allows you to customize DevTools to your preferred workflow.
 
-![The More tools menu in Focus Mode](media/focus-mode-more-tools.png)
-
 Click **More tools** (**+**) to list all the available tools. Selecting a tool will pin it to the **Activity Bar**. It will be shown by default every time DevTools is opened.
 
+![The More tools menu in Focus Mode](./focus-mode-images/more-tools.png)
+
 If there isn't enough space in the **Activity Bar** to display all pinned tools, some tools may overflow into the **More tools** menu:
 
-![The More tools menu in Focus Mode when the window is shorter, causing some of the icons to be displayed in the menu rather than the activity bar](media/focus-mode-overflow-tools.png)
+![The More tools menu in Focus Mode when the window is shorter, causing some of the icons to be displayed in the menu rather than the activity bar](./focus-mode-images/overflow-tools.png)
```

Comment:
move the alt text up (and integrate it), to become a lead-in that's presented to everyone, and then have a short-as-possible alt-text, knowing that the screen reader already read aloud the body lead-in text.